doc/source/ray-observability/monitoring-debugging/gotchas.rst

```diff
@@ -17,7 +17,7 @@ if the cluster was started previously.
 
 **Example**: If you have a file ``baz.py`` in the directory you are running Ray in, and you run the following command:
 
-.. literalinclude:: ../doc_code/gotchas.py
```

Comment:
@rkooo567 if we do move all these files, we have to set up a ton of redirects (again). not against that per se, but are we sure we keep these files there? if not, I'd suggest to just leave the files themselves where they are on master, and just modify the `_toc.yml`.

Reply:
I think it must be okay. When we reference other pages, we do by specifying the name instead of the path. I think this doc code is the only part that has issues, and I belive I fixed all issues.

Reply:
btw @maxpumperla Do we have instructions on how to set up redirects? We will have doc restructures in the future and want to learn how we should do it.